### ISSUE TITLE
Insurance policy display even after making it inactive

### ISSUE THREAD

**Issue by nishaSussol** [openmsupply/mobile#2523]
## Describe the bug
After an insurance policy has been made inactive, it still shows up in policies list in dispensary

### To reproduce
1. First, make any of the existing insurance policy inactive in the desktop, and Sync.
2. Now, in mobile, Go to Dispensary.
3. Select a patient. Choose a prescriber and select any items from the list and click next.
4. Select the insurance policy tab.
5. See error : the inactive insurance policy still shows up in the list.

### Expected behaviour
The inactive insurance policy should not show up in the list.

### Version and device info
- Tablet model: 4.0.1

### Additional context
 found while testing #2481

**Comment by joshxg** [comment]
awesome thanks @nishaSussol !

**Comment by wlthomson** [comment]
@joshxg Think this might be best suited to a `4.0.X` patch? Thoughts?

**Comment by joshxg** [comment]
@wlthomson yeah probably! It's a pretty big mess up.. although possibly slightly different than described and might need more thought. Since you can only view insurance policies from the prescription window, you need to show the inactive ones - but i'm pretty sure that inactive ones are still applying their discount

**Comment by wlthomson** [comment]
@joshxg Hm yeah, on desktop the inactive ones aren't shown in the dispensing window. I see what you mean by that being the only way to view them on mobile. But I think in the immediate term its probably best to just not display inactive policies for new prescriptions?

**Comment by joshxg** [comment]
@wlthomson thats the same as deleting them.

 I think allowing selection and having text saying "inactive" and not applying the discount is also easy and less restrictive so that's my vote!

**Comment by wlthomson** [comment]
@joshxg Not quite the same, as they can still be re-enabled on desktop. I'm just thinking about keeping consistent with desktop as much as possible, as users might get confused if the behaviour is different.

I still think your suggestion is a good option to go for, but maybe with some additional feedback, e.g. a popup warning? 

TLDR; let's go ahead as you suggest.

**Comment by joshxg** [comment]
I believe you can only edit insurance policies on the patients home store. But maybe i'm wrong. Maybe longer term - need to be able to view/edit insurance policies from the main dispensary window

**Comment by wlthomson** [comment]
@joshxg Agreed 👍 .

**Comment by wlthomson** [comment]
Fixed by #2523.

**Comment by nishaSussol** [comment]
## TESTING
Tested in mobile version 4.0.3

### Tests
- [x] When viewing an inactive policy in the dropdown, `"(inactive)"` is appended to the policy name. **-PASS-**
- [x] If an inactive policy is selected, the insurance discount rate is `"0%"`. **-PASS-**
- [x] If an inactive policy is selected, the insurance discount amount is `"0.00"`. **-PASS-**
- [x] If an inactive policy is selected, the insurance discount is not applied to the subtotal or total. **-PASS-**
